pavelib/paver_tests/utils.py

```diff
@@ -58,3 +61,52 @@ def info(self, message, *args):
             output = message
         if not output.startswith("--->"):
             self.messages.append(unicode(output))
+
+
+class CustomShMock(object):
```

Comment:
Ah, I mainly made it a class as a logical grouping. Perhaps it's better to make a custom_sh_mock.py file as a util?